pkgs/love2d/example/assets/love_example_browser/examples/015_image_rotation.lua

```
@@ -0,0 +1,16 @@
+-- Example: Rotating images
+
+angle = 0
```

Luacheck (1.2.0)

[warning] setting non-standard global variable 'angle'

(W111)